pycodestyle.py

```diff
@@ -871,17 +871,15 @@ def whitespace_around_named_parameter_equals(logical_line, tokens):
     Don't use spaces around the '=' sign when used to indicate a
     keyword argument or a default parameter value.
 
-    Okay: def complex(real, imag=0.0):
-    Okay: return magic(r=real, i=imag)
+    Okay: def complex(real, imag=0.0):\n    return magic(r=real, i=imag)
```

Comment:
I am suggesting 2 separate test cases, each of them corresponding to a piece of valid Python code.